.github/workflows/pull-request.yml

```
@@ -0,0 +1,103 @@
+        with:
+          distribution: "zulu"
+          java-version: "21"
+          cache: "gradle"
+
+      - uses: android-actions/setup-android@v3
+        with:
+          accept-android-sdk-licenses: true
+          log-accepted-android-sdk-licenses: false
+
+      - uses: apple-actions/import-codesign-certs@v5
+        with: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)